routeHandler.py

```diff
@@ -0,0 +1,64 @@
+import json
+import pymysql
+
+connection = pymysql.connect(
+    host='yba-database.c30igyqguxod.us-west-1.rds.amazonaws.com', 
+    user='admin', 
+    password='LemonsPoke9264', 
+    database='yba'
+)
```

Comment:
Never leave passwords & usernames in plaintext. I suggest you look into fetching this information from AWS SecretsManager